templates/frontend/objects/announcement_summary.tpl

```diff
@@ -9,10 +9,11 @@
  *
  * @uses $announcement Announcement The announcement to display
  *}
-
 <article class="obj_announcement_summary">
 	<h4>
-		{$announcement->getLocalizedTitle()}
+		<a href="{url router=$smarty.const.ROUTE_PAGE page="announcement" op="view" path=$announcement->getId()}">
+			{$announcement->getLocalizedTitle()}
```

Comment:
This needs the `|escape` modifier. (Announcement titles shouldn't support HTML.)